marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/ModelController.java

```diff
@@ -253,4 +268,9 @@ private ModelsService newService() {
     public static class UpdateModelInfoInput {
         public String description;
     }
+
+    public static class ModelReferencesInfo {
+        public List<String> steps;
+        public List<String> models;
```

Comment:
Since the "models" list contains entity names, this should really be called "entityNames", not "models". My assumption is that if something is called "models", it's a list of JSON entity models.